app/views/finders/_facet_collection.html.erb

```diff
@@ -56,10 +56,10 @@
             Clear all filters
           </button>
         </div>
-        <div class="facets__footer">
-          <button class="gem-c-button govuk-button js-close-filters js-show-results" type="button">
-            Show <span class="js-result-count"> <%= result_set_presenter.displayed_total %></span>
-          </button>
+        <div class="facets__footer js-close-filters js-show-results">
```

Comment:
Just wondering why the `js-` JavaScript hook class has moved from the `button` to the `div`?

Reply:
I've moved this because there's no way to get the js class onto the button component on line 60. The js for mobile filters is reliant on that class or at least some sort of hook into the page so I've amended it slightly so that a div containing the generated button markup interacts with the js.

Reply:
Is this relevant? https://components.publishing.service.gov.uk/component-guide/button/with_js_classes

Reply:
I missed that! Whoops, will sort out.